api/base/exceptions.py

```diff
@@ -26,28 +46,33 @@ def json_api_exception_handler(exc, context):
                 }
             ])
         elif isinstance(message, dict):
-            for error_key, error_description in message.iteritems():
-                if error_key in top_level_error_keys:
-                    errors.append({error_key: error_description})
-                elif error_key in resource_object_identifiers:
-                    if isinstance(error_description, basestring):
-                        error_description = [error_description]
-                    errors.extend([{'source': {'pointer': '/data/' + error_key}, 'detail': reason} for reason in error_description])
-                elif error_key == 'attributes':
-                    if isinstance(error_description, list):
-                        errors.extend([{'source': {'pointer': '/data/' + error_key}, 'detail': reason} for reason in error_description])
-                else:
-                    if isinstance(error_description, basestring):
-                        error_description = [error_description]
-                    errors.extend([{'source': {'pointer': '/data/attributes/' + error_key}, 'detail': reason} for reason in error_description])
-
+            dict_error_formatting(errors, message)
```

Comment:
What kinds of exceptions does this usually include? I'm assuming ones raised by DRF or DRF-bulk internally?